pajbot/apiwrappers/ffz.py

```diff
@@ -50,8 +50,7 @@ def fetch_channel_emotes(self, channel_name):
             if e.response.status_code == 404:
                 # user does not have any FFZ emotes
                 return []
-            else:
-                raise e
+            raise e
```

Comment:
Let's add a little space
```suggestion

            raise e
```